schemas/quality_performance_score.yaml

```diff
@@ -0,0 +1,56 @@
+---
+quality_performance_score:
+  anyOf:
+  - type: object
+    title: error
```

Comment:
how important is to have these errors described in result schema? so far I don't see any of endpoints explicitly defining error responses.

Reply:
Hard to say, the endpoint is for Orchestrator and I have no clue what kind of documentation they need or what is exactly generated by these yamls. 

Since there are errors I can document easily I decided to document them.